src/autoformat.js

```diff
@@ -27,6 +27,20 @@ export default class Autoformat extends Plugin {
 		return 'Autoformat';
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		/**
+		 * Specifies whether autoformatting is enabled or not.
+		 *
+		 * @type {Boolean}
+		 */
+		this.isEnabled = true;
```

Comment:
I do not like this change. I think that autoformat should check if commands are enabled instead.

Reply:
While rewriting this feature to use commands only (researched in #7) seems impossible now, using commands to check whether a certain autoformat feature can be executed might indeed be feasible now.